ZConfig/tests/support.py

```diff
@@ -21,21 +21,12 @@
 from ZConfig.loader import ConfigLoader
 from ZConfig.url import urljoin
 
-try:
-    import StringIO
-except ImportError:
-    # Python 3 support.
-    import io as StringIO
-
-try:
-    from urllib import pathname2url
-except ImportError:
-    # Python 3 support.
-    from urllib.request import pathname2url
+from ZConfig._compat import NStringIO as StringIO
+from ZConfig._compat import pathname2url
 
 try:
     __file__
-except NameError:
+except NameError: # pragma: no cover
     import sys
     __file__ = sys.argv[0]
```

Comment:
I vaguely recall this sort of caution from Python 2.2/2.3 timeframe, but is it still necessary?  Under what circumstances can `__file__` be undefined in this day and age?